.github/workflows/publish-to-github.yaml

```
@@ -0,0 +1,44 @@
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4
+        with:
+          artifact_name: 'published-site'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)